### DISCUSSION TITLE
Special attributes!

### DISCUSSION THREAD

**Discussion by CaptainCat999** [Second-Hand-Friends/kleinanzeigen-bot#173] [Q&A]
Hello,

First of all, thank you for creating a great bot!
I configured pretty much everything, until I stumbled upon a timeout error. Why? Because I'm publishing an ad in the foreign real estate category (I found that it was under 195-275 code on eBay website), BUT there is an attribute that must be filled out for the ad to be published.

This field is called: Lage (position) and in the drop-down menu I need to choose "Ausland". Without it the ad is not published and the bot is helpless!
Is it something programmable? It's not a special attribute, imho, but an additional menu to choose from in the ad.
There are also drop-down menus like Year/Month availability, and a series of options to tick like  "Keller", "TV". Is it possible to code it in?

I send a screenshot of the fields to fill out!

Thanks a lot in advance!
CaptainCat

![ebay ad immo](https://user-images.githubusercontent.com/131389883/233438232-8793d5c6-c13a-433a-a859-7181ffe2a681.png)

**Comment by CaptainCat999** [comment]
@sebthom thanks in advance!

**Reply by CaptainCat999** [reply]
Anyone?

**Reply by alonzomarino33512A** [reply]
Hello, you descover from where was the problem ? Thanks
